### PR TITLE
added erc 721 example.

### PR DESCRIPTION
**Pull request by yenkhoon**
## Description

Added erc 721 example

## Related issue(s)
#44

## Checklist

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] CHANGELOG is updated.

### CODE REVIEW

**Comment by yenkhoon** [comment]
don't merge yet i am fixing some error.

**Comment by yenkhoon** [comment]
burn function have not supported as issue #51

File: examples/erc721/src/contract.rs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+instantiate creates a new contract and deployment code.
+*/
+#[kelk_entry]
+pub fn instantiate(ctx: Context, msg: InstantiateMsg) -> Result<(), Error> {
+    ERC721::instantiate(ctx, &msg.name, &msg.symbol)?;
+    Ok(())
+}
+
+/*
+process executes the contract associated with the addr with the given input as
```
Comment (b00f) [inline examples/erc721/src/contract.rs:17]:
```suggestion
process executes the contract with the given `msg` as
```

File: examples/erc721/src/contract.rs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        ProcMsg::ApprovedForAll { operator, approved } => {
+            token.set_approval_for_all(operator, approved)
+        }
+        ProcMsg::Mint { addr, amount } => token.mint(addr, amount),
+        ProcMsg::Burn { addr, amount } => token.burn(addr, amount),
+    }
+}
+
+/*
+query executes the contract associated with the addr with the given input
```
Comment (b00f) [inline examples/erc721/src/contract.rs:38]:
```suggestion
query executes the contract with the given `msg`
```

File: examples/erc721/src/erc721.rs [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+    pub fn name(&self) -> Result<String, Error> {
+        Ok(self.name.get_string()?)
+    }
+
+    pub fn symbol(&self) -> Result<String, Error> {
+        Ok(self.symbol.get_string()?)
+    }
+
+    pub fn balance_of(&self, addr: &Address) -> Result<i64, Error> {
+        let balance = self.balances.find(addr).unwrap().unwrap_or(0);
```
Comment (b00f) [inline examples/erc721/src/erc721.rs:102]:
```suggestion
        let balance = self.balances.find(addr)?.unwrap_or(0);
```

File: examples/erc721/src/erc721.rs [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+        Ok(balance)
+    }
+
+    pub fn owner_of(&self, token_id: &i64) -> Result<Address, Error> {
+        let owner = self.owners.find(token_id).unwrap();
```
Comment (b00f) [inline examples/erc721/src/erc721.rs:107]:
```suggestion
        let owner = self.owners.find(token_id)?;
```

**Review by b00f — CHANGES_REQUESTED** [review]

**Review by b00f — APPROVED** [review]

**Pull request merged**